examples/vector_database/compute_vectors.yaml

```
@@ -0,0 +1,34 @@
+name: clip-batch-compute-vectors
+
+resources:
+  accelerators: {L4:1, A10G:1, A10:1, V100:1}
+  memory: 32+
+
+num_nodes: 1
+
+file_mounts:
+  /output:
+    name: sky-clip-embedding
+    mode: MOUNT
+
+envs:
+  HF_TOKEN: ${HF_TOKEN}
+  START_IDX: ${START_IDX}
+  END_IDX: ${END_IDX}
```

```suggestion
envs:
  HF_TOKEN: ''
  START_IDX: ''
  END_IDX: ''
```
Maybe this will work instead